programs/mpl-core/src/plugins/external_plugins.rs

```diff
@@ -273,6 +278,113 @@ pub enum ExtraAccount {
     },
 }
 
+impl ExtraAccount {
+    pub(crate) fn derive(
+        &self,
+        program_id: &Pubkey,
```

Comment:
Shouldn't this be crate::ID always? Otherwise what's the point of is_signer?

Reply:
Discussed offline.  Leaving it as passed-in program ID.